barretenberg/cpp/src/barretenberg/circuit_checker/mega_circuit_builder.test.cpp

```diff
@@ -158,4 +159,44 @@ TEST(MegaCircuitBuilder, GoblinEccOpQueueUltraOps)
     }
 }
 
+TEST(MegaCircuitBuilder, CompleteSelectorPartitioningCheck)
```

Comment:
Would be helpful to say in one sentence what this test is testing.

Reply:
done